src/util/Params.cpp

```diff
@@ -396,6 +396,8 @@ Params::Params() : finalized_(false) {
   InitStringParam("output-dir", "crux-output",
     "The name of the directory where output files will be created.",
     "Available for most commands.", true);
+  InitStringParam("spectrum-outdir", "crux-output", "The name of the directory where the converted spectrum files will be stored. "
```

Comment:
This does not work as intened. This will appear as: ```[--spectrum-outdir <string>]
     The name of the directory where the converted spectrum files will be
     stored. Default = --output-dir value Default = crux-output.```
     I think it should be 
     ```InitStringParam("spectrum-outdir", "--output-dir valuet", "The name of the dire...```